homeassistant/components/broadlink/sensor.py

```diff
@@ -48,68 +18,52 @@
     "noise": ["Noise", " "],
 }
 
-DEVICE_TYPES = A1_TYPES + RM_TYPES + RM4_TYPES
-
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): vol.Coerce(str),
-        vol.Optional(CONF_MONITORED_CONDITIONS, default=[]): vol.All(
-            cv.ensure_list, [vol.In(SENSOR_TYPES)]
-        ),
-        vol.Required(CONF_HOST): vol.All(vol.Any(hostname, ip_address), cv.string),
-        vol.Required(CONF_MAC): mac_address,
-        vol.Optional(CONF_TYPE, default=DEVICE_TYPES[0]): vol.In(DEVICE_TYPES),
-        vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.positive_int,
-    }
+    {vol.Required(CONF_HOST): cv.string}, extra=vol.ALLOW_EXTRA
```

Comment:
We should keep the legacy platform schemas intact, to avoid breaking the import flow, but add a deprecation message, saying that the user config yaml has been imported and the user can remove the config yaml.

Reply:
I am allowing extra keys, so I get exactly what we need (the host) and ignore the rest. Things won't break. This makes the code cleaner, otherwise I will have to import all those deprecated constants and defaults on their related platforms.

I think the depreciation notice is important, I'll do it. How about this message?

`The remote platform is deprecated. To configure your device, click Settings in the sidebar and click Integrations. If you see your device, click Configure, otherwise, click + and enter the host. Follow the instructions to complete the setup and delete your configuration for this platform when you are done.`

Reply:
The switch configuration will break. There is not much I can do without adding a considerable amount of logic to this code.